src/RA_AchievementSet.h

```diff
@@ -4,6 +4,10 @@
 
 #include "RA_Achievement.h" // RA_Condition.h (RA_Defs.h)
 
+#if !defined(RAPIDJSON_DOCUMENT_H_) && defined(RA_UTEST)
+#define RAPIDJSON_NOMEMBERITERATORCLASS 1
+#include <rapidjson\include\rapidjson\document.h>
+#endif // !!defined(RAPIDJSON_DOCUMENT_H_) && defined(RA_UTEST)
```

Comment:
Unit test wouldn't compile, probably because of the `RA_EXPORTS `macro.

Reply:
Preferable alternative is to exclude `OnRequestUnlocks` from the unit test compile:
```
#ifndef RA_UTEST
    static void OnRequestUnlocks(const Document& doc);
#endif
```
A similar change was made for `Achievement::Parse` in RA_Achievement.h

Reply:
Seems like a good approach to me.